x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/TestCaseSupplier.java

```diff
@@ -1435,7 +1435,7 @@ public static TestCase typeError(List<TypedData> data, String expectedTypeError)
             this.source = Source.EMPTY;
             this.data = data;
             this.evaluatorToString = evaluatorToString;
-            this.expectedType = expectedType;
+            this.expectedType = expectedType == null ? null : expectedType.noText();
```

Comment:
This is the comprehensive 'catch all' case we added late in the PR once we had covered all the low hanging fruit. But perhaps this means we can remove some of the other earlier code. I'll investigate.

Reply:
Indeed there were 12 places we had added this `noText()` that no longer needed it, since they all called into this TestCase constructor. I've removed them all and will push a commit after re-running tests to be safe.